docs/docs/02-quick-start/01-installation.md

```diff
@@ -10,19 +10,52 @@ go install github.com/a-h/templ/cmd/templ@latest
 
 This installs templ into your path.
 
-## go install (as tool)
+## go install (as tool) - Recommended for projects
 
-To install templ locally in your project, run:
+:::tip Recommended approach
+The tools directive introduced in Go 1.24 is the recommended way to manage templ in your projects, as it ensures version consistency between the generator and runtime.
+:::
+
+To install templ as a tool in your project, run:
 
 ```bash
 go get -tool github.com/a-h/templ/cmd/templ@latest
 ```
 
-:::info 
-This uses the [tool directive](https://tip.golang.org/doc/modules/managing-dependencies#tools) feature of Go added in v1.24. 
+This approach provides several advantages:
 
-To run templ once installed, use `go tool templ` instead of `templ`.
-:::
+- **Version consistency**: Links the generator version to the runtime version in your `go.mod`
```

Comment:
This sounds like the same point as the next one?

I don't like the `**Something**: something else` pattern really. It forces you to try and say "something else" after your main point.

I don't think "clean" environment means anything.